cmd/collector/main.go

```diff
@@ -131,6 +144,110 @@ func main() {
 	}
 }
 
+func retrieveVlanInformation(logger *log.Logger, govmomiClient *govmomi.Client) []string {
+	logger.Println("retrieveVlanInformation")
+	ctx := context.Background()
+
+	seenVlanIds := retrieveDistributedVLANs(logger, govmomiClient, ctx)
+	seenVlanIds = retrieveStandardVLANs(logger, govmomiClient, ctx, seenVlanIds)
+	for
+	return vlanIDs
+}
+
+func retrieveStandardVLANs(logger *log.Logger, govmomiClient *govmomi.Client, ctx context.Context, seenVlanIds map[string]bool) map[string]bool {
+	// Use ViewManager to create a view of HostSystem objects
+	m := view.NewManager(govmomiClient.Client)
+	v, err := m.CreateContainerView(ctx, govmomiClient.ServiceContent.RootFolder, []string{"HostSystem"}, true)
+	if err != nil {
+		logger.Fatal("Error creating container view: %v\n", err)
+		return seenVlanIds
+	}
+	defer v.Destroy(ctx)
+
+	// Retrieve all HostSystem objects
+	var hostList []mo.HostSystem
+	err = v.Retrieve(ctx, []string{"HostSystem"}, []string{"name", "config.network.vswitch"}, &hostList)
```

Comment:
I think MTV fetches those information as well:
https://github.com/kubev2v/forklift/blob/main/pkg/controller/provider/container/vsphere/collector.go#L75

It's not possible to reuse? Do we need to retrieve it by ourselfs?

Reply:
https://github.com/kubev2v/forklift/blob/main/pkg/controller/provider/model/vsphere/model.go#L204